.github/workflows/ci.yml

```diff
@@ -80,15 +80,18 @@ jobs:
         git fetch upstream
         if git diff upstream/master --name-only | grep -q "^asv_bench/"; then
             asv machine --yes
-            ASV_OUTPUT="$(asv dev)"
-            if [[ $(echo "$ASV_OUTPUT" | grep "failed") ]]; then
-                echo "##vso[task.logissue type=error]Benchmarks run with errors"
-                echo "$ASV_OUTPUT"
+            asv dev | sed "/failed$/ s/^/##[error]/" | tee benchmarks.log
```

Comment:
Is there a reason for using `tee` here instead of simply redirecting the `sed` output?

Reply:
I want the log to be shown in the CI, tee gets it printed (besides saving the file later used to see if there are failures and to save the artifact)